src/components/Tasks/TaskListItem.tsx

```diff
@@ -36,26 +36,40 @@ interface PropsType {
   entry: TaskType;
   onClick: (task: TaskType) => void;
   onSave: (item: PimType, journalUid: string, originalItem?: PimType) => Promise<void>;
+  onUpdate: (message: string) => void;
 }
 
 export default React.memo(function TaskListItem(props: PropsType) {
   const {
     entry: task,
     onClick,
     onSave: save,
+    onUpdate: update,
```

Comment:
Why rename this function?

Reply:
I think I just wanted to keep it consistent, but I've removed these functions now